system/core/pages.php

```diff
@@ -242,6 +305,9 @@ function loadPage(RequestContext $O_O) {
 						break;
 				}
 				break;
+			case HyphaRequest::HYPHA_SYSTEM_PAGE_TAG_INDEX:
+				$hyphaPage = new tagindexpage($O_O);
```

Comment:
I notice that this function is not camel cased. (as a very minor non blocking review suggestion :-) )

Reply:
The datatype class names use lowercase naming. Maybe we should change that at some point, but for now I'm just staying consistent with the existing classes.